src/custom/api/gnosisProtocol/errors/OperatorError.ts

```diff
@@ -6,21 +6,27 @@ export interface ApiErrorObject {
 }
 
 // Conforms to backend API
-// https://github.com/gnosis/gp-v2-services/blob/0bd5f7743bebaa5acd3be13e35ede2326a096f14/orderbook/openapi.yml#L562
+// https://github.com/gnosis/gp-v2-services/blob/d932e11c9a2125fdba239530be7684799f694909/crates/orderbook/openapi.yml#L801
+// and
+// https://github.com/gnosis/gp-v2-services/blob/d932e11c9a2125fdba239530be7684799f694909/crates/orderbook/openapi.yml#L740
```

Comment:
Nice you add the blob. As a nitpick, I would also make a comment with something: See latest version and a link to their main branch file where they always have the latest

This way we can easily check if there's sth new